src/desktop/src/libs/SeedStore/SeedStoreCore.js

```diff
@@ -41,4 +41,39 @@ export default class SeedStoreCore {
     getDigest(trytes) {
         return Promise.resolve(iota.utils.transactionObject(trytes).hash);
     }
+
+    /**
+     * Mines bundle
+     *
+     * @method mineBundle
+     *
+     * @param {Int8Array} normalizedBundle
+     * @param {number} numberOfFragments
+     * @param {Int8Array} bundleEssence
+     * @param {number} [essenceLength]
+     * @param {number} [count]
+     * @param {number} [procs]
+     * @param {number} [miningThreshold]
+     *
+     * @returns {Promise<number>}
+     */
+    mineBundle(
+        normalizedBundle,
+        bundleEssence,
+        security = 2,
```

Comment:
Should these parameters be reversed? `security` is before `essence` here: https://github.com/iotaledger/trinity-wallet/pull/2226/files#diff-bb2d74f9facbddd9e9cd67eef7eccfc8R73

Reply:
Nope, security is used to get relevant part of aggregated norm. bundle